build.gradle

```diff
@@ -24,16 +30,79 @@ dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
     antlr 'org.antlr:antlr4:4.5'
 
-    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
-    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
+    // using groovy 2.3 to avoid problems in eclipse
+    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
```

Comment:
On Mon, Aug 03, 2015 at 04:00:37PM -0700, Jakub Jirutka wrote:
> 
>    In [1]build.gradle:
> > @@ -24,16 +30,79 @@ dependencies {
> >      compile 'org.slf4j:slf4j-api:1.7.12'
> >      antlr 'org.antlr:antlr4:4.5'
> >
> > -    testCompile 'org.codehaus.groovy:groovy-all:2.4.3'
> > -    testCompile 'org.spockframework:spock-core:1.0-groovy-2.4'
> > +    // using groovy 2.3 to avoid problems in eclipse
> > +    testCompile 'org.codehaus.groovy:groovy-all:2.3.11'
> 
>    I think that those people haver never used any other IDEâ¦ :smirk_cat:
>    Eclipse is the worst IDE Iâve ever seen, itâs so extremely buggy that
>    itâs hard to distinguish whatâs a bad feature and whatâs a bug.

You're entitled to your opinion, even if it's wrong. In fact I've used more IDEs
than there are versions of Java SE.
 
>    Anyway, the point is that we should IMHO never take any decision just
>    because of IDE xyz doesnât work properlyâ¦

On that we agree.